xarray/tests/test_array_api.py

```diff
@@ -0,0 +1,48 @@
+import numpy.array_api as xp
```

Comment:
```suggestion
from numpy import array_api as xp
```

Reply:
Yes, this was a mistake - I've changed the array API import to be below `importerskip`, and added `isort:skip`.

Thanks for the suggestions.